packages/perseus/src/widgets/interactive-graphs/interactive-graph-question-builder.ts

```diff
@@ -252,6 +258,20 @@ class InteractiveGraphQuestionBuilder {
         return this;
     }
 
+    addLockedFunction(equation: string, options?: LockedFunctionOptions) {
+        const lockedFunction: LockedFunctionType = {
+            type: "function",
+            equation,
+            color: "grayH",
+            strokeStyle: "solid",
+            directionalAxis: "x",
+            ...options,
+        };
+
+        this.addLockedFigure(lockedFunction);
+        return this;
+    }
+
```

Comment:
Don't forget to add a unit test for this in `interactive-graph-question-builder.test.ts`!

Reply:
Good catch. Thanks.

Added.